### DISCUSSION TITLE
SMTP Settings would be wonderful

### DISCUSSION THREAD

**Discussion by Leopere** [JordanKnott/taskcafe#147] [Ideas]
Just to be able to receive update emails may be keen.  Hard to implement in a way that isn't exactly spammy but really awesome if possible.

**Comment by crazyyzarc** [comment]
It should actually be possible to set it in the configuration
https://github.com/JordanKnott/taskcafe/blob/4f5aa2deb80aadf87affce728378bc89823211e0/conf/taskcafe.example.toml

**Reply by JonLittleIT** [reply]
@crazyyzarc did you get this setup?

**Reply by Leopere** [reply]
ended up bailing a while back after hearing about a complete refactor I like the software but couldn't invest in something that was about to completely change at the center of the workflow.  Thanks for your efforts though!
